### PR TITLE
Add L4C as convenience DOI value

### LINKED ISSUE #90: Add L4C "convenience" DOI
There is a new L4C GEDI collection with DOI 10.3334/ORNLDAAC/2338. Users are able to specify "10.3334/ORNLDAAC/2338" as the value for the `doi` input, but as a convenience, we want to allow them to use "L4C" as the value for the `doi` input.

### PR DESCRIPTION
**Pull request by chuckwondo**
Also, update to MAAP base image to streamline registration.

Fixes #90

### CODE REVIEW

File: bin/algo/delete [hunk 1]
```diff
@@ -39,7 +39,7 @@ branch=$(git rev-parse --abbrev-ref HEAD)
 algorithm_name=$(grep "^algorithm_name:" "${yaml_file}" | sed -E 's/algorithm_name: ([^[:space:]]+).*/\1/')
 algorithm_version=$(grep "^algorithm_version:" "${yaml_file}" | sed -E 's/algorithm_version: ([^[:space:]]+).*/\1/')
 [[ "${branch}" != "main" ]] && algorithm_version=${branch}
-algorithm_id="${algorithm_name}:${algorithm_version}"
+algorithm_id="${algorithm_name}:${VERSION:-${algorithm_version}}"
```
Comment (jjfrench) [inline bin/algo/delete:42]: Where VERSION would be something you pass through the command line rather than using the default algorithm_version?

Comment (chuckwondo) [reply bin/algo/delete:42]: Exactly.

File: algorithm_config.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
-algorithm_description: Subset GEDI L1B, L2A, L2B, or L4A granules within an area of interest (AOI)
+algorithm_description: Subset GEDI L1B, L2A, L2B, L4A, or L4C granules within an area of interest (AOI)
 algorithm_name: gedi-subset
-algorithm_version: 0.8.0
+algorithm_version: 0.9.0
 repository_url: https://github.com/MAAP-Project/gedi-subsetter.git
-docker_container_url: mas.maap-project.org/root/maap-workspaces/base_images/python:v4.0.0
+docker_container_url: mas.maap-project.org/root/maap-workspaces/custom_images/maap_base:v4.1.0
```
Comment (jjfrench) [inline algorithm_config.yaml:5]: Is this the new default registration image? Should ask JPL to update the value in the UI?

**Review by jjfrench — APPROVED** [review]
Changes look good, thanks!

File: algorithm_config.yaml (hunk 2 above)
Comment (chuckwondo) [reply algorithm_config.yaml:5]: I don't know if it's the default, nor if it should be. This is the newly added, most basic, stripped down image, which is what we want for GEDI Subsetter, but I don't know how many other algorithms would make use of it.

**Pull request merged**